src/app/component/auth/components/sign-up/sign-up.component.ts

```diff
@@ -188,4 +184,10 @@ export class SignUpComponent implements OnInit {
       this.passwordConfirmErrorMessageBackEnd = error.name === 'password' ? error.message : this.passwordConfirmErrorMessageBackEnd;
     });
   }
+
+  ngOnDestroy(): void {
+    if (this.subscriptions.length !== 0 ) {
```

Comment:
private destroy = new Subject();

    takeUntil(this.destroy)

    ngOnDestroy(): void {
        this.destroy.next(true);
        this.destroy.complete();
    }

Reply:
thanks, fixed